skills_asses/src/app/login/login.component.html

```diff
@@ -0,0 +1,47 @@
+
+<div id="container">
+    <section class="container-fluid" id="left">
+        <div id="login">
+            <div id="title">
+                <h1>LOGIN</h1>
+                <p>
+                    Please enter your credintials
+                </p>
+            </div>
+            <div id ="errormsg"
+                *ngIf="submitted && (loginForm.controls.username.invalid || loginForm.controls.password.invalid)"
+                >
+                <div
+                *ngIf="loginForm.controls.username.errors?.['required']
+                || loginForm.controls.password.errors?.['required'] || loginForm.controls.password.errors?.['minlength']">
+                    Please enter a correct national ID/password
+                </div>
+                <div
+                *ngIf="loginForm.controls.username.errors?.['minlength'] || loginForm.controls.username.errors?.['maxlength']">
+                    Username should be 10 numbers.
```

Comment:
I recommend moving message content/all labels to a file, we normally use this library: https://www.npmjs.com/package/@ngx-translate/core